sydent/terms/terms.py

```diff
@@ -107,7 +107,7 @@ def get_terms(sydent) -> Optional[Terms]:
             return Terms(None)
 
         with open(termsPath) as fp:
-            termsYaml = yaml.full_load(fp)
+            termsYaml = yaml.safe_load(fp)
```

Comment:
Why this change? (it seems unrelated)

Reply:
Oh shoot you are correct-this was a change to fix an upstream api change that got rid of `full_load`, thus blowing up the CI. I'll remove it out of this PR and open another one for this change.

Reply:
Hmm that is super weird, using 'full_load' blows up all the tests relating to `terms` on my machine, I wonder why not in github actions. Regardless, this change has been removed.